worker/chainbusservice.go

```diff
@@ -179,12 +191,22 @@ func (bs *BusService) requestLastBlock() (
 
 // RequestSQLProfile get specified database profile.
 func (bs *BusService) RequestSQLProfile(dbID proto.DatabaseID) (p *types.SQLChainProfile, ok bool) {
-	bs.lock.Lock()
-	defer bs.lock.Unlock()
+	bs.lock.RLock()
+	defer bs.lock.RUnlock()
 	p, ok = bs.sqlChainProfiles[dbID]
 	return
 }
 
+func (bs *BusService) RequestPermStat(dbID proto.DatabaseID, user proto.AccountAddress) (permStat *types.PermStat, ok bool) {
```

Comment:
**[golint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
exported method BusService.RequestPermStat should have comment or be unexported